### DISCUSSION TITLE
Suggestion: Automatic extra disk

### DISCUSSION THREAD

**Discussion by FreeLANMan** [fdivitto/FabGL#271] [Ideas]
Suggestion: automatically add a 20MB HD to the system (30MB would work?), for data and keep the rest the same, then the user chooses 1 disk and has two to use.

**Comment by fdivitto** [comment]
do you mean a disk "D" (or C in case of boot from floppy)?

**Comment by FreeLANMan** [comment]
Sorry, I hadn't seen that I already had the option to add more disks...
But one thing that I find very interesting is to give the option to boot from a floppy image in the internal memory, which would give the option to use the system without a microSD card.

**Comment by fdivitto** [comment]
>>> to boot from a floppy image in the internal memory, which would give the option to use the system without a microSD card.

yes, this is very interesting! I will try to implement it.

**Comment by FreeLANMan** [comment]
I think that would have to ask the user if he wants to boot from the internal memory and give a limit of 5 seconds, in case the system finds a floppy image in the internal memory. And use a default name for the floppy image that the system will look for.
